libRALFit/test/example_module.f90

```diff
@@ -2178,20 +2178,20 @@ subroutine solve_LLS_tests(options,fails)
         fails = fails + 1
      end if
 
-
-     n = 100
-     m = 20
+      n = 65
+      m = 60
      deallocate(J,f,Jd,d)
      allocate(J(n*m), f(m), d(n)) ! f has wrong size
 
      call setup_workspaces(work,n,m,options,status)
 
      J = 1.0_wp
+     J(1:m) = 0.0_wp
```

Comment:
@tyronerees critical change: Make a row of zeros to promote rank-deficiency detection, this is required because under certain compilers `dtrtrs` would not detect degeneracy even though the resulting solve is rubbish.